OracleDatabase/SingleInstance/README.md

```diff
@@ -239,17 +239,17 @@ To configure  wallet password, please use the following command:
         # ORACLE_SID default value is ORCLCDB
         docker cp <container name>:/opt/oracle/oradata/clientWallet/<ORACLE_SID> <destination directory>
 
-* The client wallet directory above will include wallet files, along with sample `sqlnet.ora` and `tnsnames.ora` files. You should edit the `HOST` and `PORT` fields accordingly in the `tnsnames.ora` before connecting using TCPS.
-* After `tnsnames.ora` is modified, go inside the downloaded client wallet directory and set TNS_ADMIN for SQL\*Plus by using the `export TNS_ADMIN=$(pwd)` command. Then users can connect via TCPS with, for example, the following commands:
+* The client wallet directory above will include wallet files, along with sample `sqlnet.ora` and `tnsnames.ora` files.
+* To connect to the database via TCPS use EZCONNECT (easy connect naming method) as shown:
 
-        # Connecting Enterprise Edition
-        sqlplus sys@ORCLCDB as sysdba
-        # Connecting Express Edition
-        sqlplus sys@XE as sysdba
+        sqlplus sys@tcps://<host>:<port>/<service_name>?wallet_location=<wallet_directory> as sysdba
+        # service_name could be ORACLE_SID or ORACLE_PDB
```

Comment:
(i) by implicit definition service_name is a Service Name, not a SID
(ii) SIDs don't work with Easy Connect syntax used so SID should be removed from the statement

What is ORACLE_PDB?  This is a non (traditional) standard variable.

Reply:
removed that comment